File: Python/Reverse Integer/ReverseInteger.py
class Solution:
    def reverse(self, x):
        """
        :type x: int
        :rtype: int
        """
        isNegative=True if x<0 else False
        strX=str(-x) if isNegative else str(x)
        prefix="-" if isNegative else ""
        result=prefix+strX[::-1]
        result=int(result)
        if(result<-2147483648 or result>2147483647):
            result=0
        return result

File: C#/Reverse Integer/ReverseInteger.cs
public class Solution {
    public int Reverse (int x) {
        bool reverse = x < 0 ? true : false;
        string strX = reverse?(-x).ToString () : x.ToString ();
        int len = strX.Length;
        StringBuilder resultStr = new StringBuilder (reverse? "-": "");
        for (int i = 0; i < len; i++) {
            resultStr.Append (strX[len - i - 1]);
        }
        int result = 0;
        int.TryParse (resultStr.ToString (), out result);
        return result;
    }
}